code/images/SCIENCE.html

```diff
@@ -0,0 +1,165 @@
+            et dolore magna aliqua. <br><button class="featured-button">Read More.......</button>  </p>
+        </div>
+        </div>
+    </main>
+    
+     <section><div class="news-section">
+       <div class="container1">
+        <a href="http://127.0.0.1:5500/code/index.html#"><h3 class="l-heading">TRENDING ARTICLES <span class="arrow">↝</span></h3>  </a>
+        
+        <div class="latest-images">
+         
+     <div class="image1"><img src="./SCIENCE_files/latest1.jfif" alt=""> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+      velit esse cillum dolore eu fugiat nulla pariatur. </p><button>Read more.... </button>
+          </div>
+         <div class="image2"><img src="./SCIENCE_files/latest2.jfif" alt=""><p>Lorem ipsum dolor sit amet, nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+          velit esse cillum dolore eu fugiat nulla pariatur. </p><button>Read more.... </button></div> 
+          
+          <div class="image3"><img src="./SCIENCE_files/latest3.jfif" alt=""><p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+            velit esse cillum dolore eu fugiat nulla pariatur. </p><button>Read more.... </button></div>
+          <div class="image4"><img src="./SCIENCE_files/latest4.jfif" alt=""><p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+            velit esse cillum dolore eu fugiat nulla pariatur.</p><button>Read more.... </button></div>
+            <div class="image5"><img src="./SCIENCE_files/latest 5.jfif" alt=""> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+              velit esse cillum dolore eu fugiat nulla pariatur. </p><button>Read more.... </button> </div>
+              <div class="image6"><img src="./SCIENCE_files/latest6.jfif" alt=""> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+                velit esse cillum dolore eu fugiat nulla pariatur. </p><button>Read more.... </button>  </div>
+                <div class="image6"><img src="./SCIENCE_files/latest7.jfif" alt=""> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elitlaboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate 
+                  velit esse cillum dolore eu fugiat nulla pariatur. </p>  <button>Read more.... </button></div>
+                  
+        
+
+
+     </div>
+    </div></div></section> 
+      
+
+ 
+  <!-- Code injected by live-server -->
```

Comment:
Is this code supposed to be in the index-file or in another js-file? Maybe both works :D

Reply:
I downloaded font-awesome in my local desktop and saved them in the folder CSS for media icons class. so I think the code comes here but it is not seen in index.html in GitHub.